src/gatsby/utils/fetchCache.ts

```diff
@@ -24,8 +24,8 @@ export function makeFetchCache<DataType>({dataFetch, name}: Options) {
 
     async function fetchData() {
       try {
-        const result = await dataFetch();
-        dataCache = result.data;
```

Comment:
was kinda mixed up before having `dataFetch` return it's response but was typed as `any`